.github/workflows/pr_stage_test.yml

```diff
@@ -0,0 +1,147 @@
+name: pr_stage_test
+
+on:
+  pull_request:
+    paths-ignore:
+      - 'README.md'
+      - 'README_zh-CN.md'
+      - 'docs/**'
+      - 'demo/**'
+      - '.dev_scripts/**'
+      - '.circleci/**'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build_cpu:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.7]
+        include:
+          - torch: 1.8.1
+            torchvision: 0.9.1
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Upgrade pip
+        run: pip install pip --upgrade
+      - name: Get MMCV_TORCH as the environment variable
+        run: . .github/workflows/scripts/get_mmcv_var.sh ${{matrix.torch}}
+        shell: bash
+      - name: Install Pillow
+        run: pip install Pillow==6.2.2
+        if: ${{matrix.torchvision == '0.4.2'}}
+      - name: Install PyTorch
+        run: pip install torch==${{matrix.torch}}+cpu torchvision==${{matrix.torchvision}}+cpu -f https://download.pytorch.org/whl/torch_stable.html
+      - name: Install MMCV
+        run: pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cpu/torch${MMCV_TORCH}/index.html
```

Comment:
please follow these lines to install mmcv and mmengine
https://github.com/open-mmlab/mmsegmentation/blob/6cdc2c4a8a4f3abc71ee8acdd52c0a46326abe4c/.github/workflows/build.yml#L58-L66